lib/generators/declare_schema/migration/migration_generator.rb

```diff
@@ -73,8 +73,8 @@ def migrate
         end ||
         default_migration_name
 
-      @up = indent(up, 4)
-      @down = indent(down, 4)
+      @up = indent(up.strip, 4)
+      @down = indent(down.strip, 4)
```

Comment:
While I was here, I decided to strip the empty lines at the end of the `up` and `down` methods since they looked a bit distracting, and hound would have complained about them.